File: app/src/main/java/com/demo/android_base/test/Mainactivity.java
package com.demo.android_base.test;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import java.io.IOException;

/**
 * created by tea9 at 2018/11/23
 */
public class Mainactivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        try {
            Runtime.getRuntime().exec("命令");
        } catch (IOException e) {
            e.printStackTrace();
        }

    }
}

File: app/src/main/java/com/demo/android_base/MainActivity.kt
package com.demo.android_base

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import com.demo.android_base.base.activity.ExpandableUseActivity
import com.demo.android_base.demo.activity.DataContrastActivity
import com.demo.android_base.demo.activity.DataContrastOtherActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), View.OnClickListener {
    override fun onClick(p0: View?) {
        when(p0!!.id){
            R.id.btn1 -> startActivity(Intent(this,ExpandableUseActivity::class.java))
            R.id.btn2 -> startActivity(Intent(this,DataContrastActivity::class.java))
            R.id.btn3 -> startActivity(Intent(this,DataContrastOtherActivity::class.java))
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        btn1.setOnClickListener(this)
        btn2.setOnClickListener(this)
        btn3.setOnClickListener(this)
    }
}
